### PR TITLE
[docs] Added molecule-plugins to "How to run tests"

### PR DESCRIPTION
**Pull request by battleboy-cxx**
molecule-plugins should also be installed, otherwise may raise `CRITICAL Failed to find driver docker. Please ensure that the driver is correctly installed.` error, meet this issue when I was trying to execute `molecule test -s local` for test purpose, https://github.com/ansible/molecule/issues/920#issuecomment-1868741469

image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -294,6 +294,7 @@ If you haven't installed docker yet, you need to install it (example for linux d
 
 **Step 5**: Run molecule test
 
+    cd openwisp.openwisp2
```
Comment (nemesifier) [inline README.md:297]: let's move this above, after the `git clone`

**Review by nemesifier — CHANGES_REQUESTED** [review]
Thanks @battleboy-cxx, the QA checks for this will fail, please run the qa checks as they're run in the build to fix the commit message format: https://github.com/openwisp/ansible-openwisp2/blob/master/.github/workflows/ci.yml#L48. See also my comment below.

The commit message could be something like:

```
[docs] Added molecule-plugins to "How to run tests"
```

File: README.md (hunk 1 above)
Comment (battleboy-cxx) [reply README.md]: Done, thanks a lot for your guiding! It's my first time to submit a PR to a open-source project. BTW, I notice it takes a lot time to run test (5 mins) when I was trying to `commit`, is it normal?

Comment (nemesifier) [reply README.md]: Yes, it's normal for this repository.

Comment (nemesifier) [reply README.md]: You do not need to add the build output in the commit message BTW, is that intentional?

Comment (battleboy-cxx) [reply README.md]: It's weird, I thought I didn't attach the build output in the commit message (maybe due to the wrong git command). Let me correct it anyway~

Comment (battleboy-cxx) [reply README.md]: Done, thanks!

**Review by nemesifier — APPROVED** [review]

**Pull request merged**